lib/glific/templates.ex

```diff
@@ -546,9 +546,15 @@ defmodule Glific.Templates do
           {:ok, SessionTemplate.t()} | {:error, Ecto.Changeset.t()}
 
   defp do_update_hsm(template, db_templates) do
+<<<<<<< Updated upstream
     current_template =
       db_templates[template["bsp_id"]]
 
+=======
+    current_template = db_templates[template["bsp_id"]]
+
+    IO.inspect(current_template)
+>>>>>>> Stashed changes
```

Comment:
Can you clean up the code, there seems to be comments from git too.
Use `mix compile` to compile the code